README.md

```diff
@@ -1,48 +1,521 @@
 # pah-fm
 
+[Fleet Manager](https://en.wikipedia.org/wiki/Fleet_management) for [Polish Humanitarian Action](https://www.pah.org.pl).
+
 [![Build Status](https://travis-ci.com/CodeForPoznan/pah-fm.svg?branch=master)](https://travis-ci.com/CodeForPoznan/pah-fm)
-[![Join the chat at https://gitter.im/CodeForPoznan/pah-fm](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/CodeForPoznan/pah-fm?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join Slack](https://img.shields.io/badge/slack-join%20chat-4a154b)](https://join.slack.com/t/codeforpoznan/shared_invite/enQtNjQ5MTU1MDI0NDA0LWNhYTA3NGQ0MmQ5ODgxODE3ODJlZjc3NWE0NTMzZjhmNDBkN2QwMzNhYWY5OWQ5MGE2OGM3NjAyODBlY2VjNjU)
```

Comment:
![obraz](https://user-images.githubusercontent.com/27100732/90830525-1baeab00-e342-11ea-9ce1-ee4f5add8999.png)
It looks like this for me, a new line after line 5 may help?

Currently, the whole element [BuildStatus/Join Slack] redirect to Slack Invite.